ember-container-query/rollup.config.mjs

```diff
@@ -10,7 +10,10 @@ const addon = new Addon({
 export default {
   // This provides defaults that work well alongside `publicEntrypoints` below.
   // You can augment this if you need to.
-  output: addon.output(),
+  output: {
+    ...addon.output(),
+    experimentalMinChunkSize: 0,
```

Comment:
Luckily, `output.experimentalMinChunkSize = 1` didn't affect `dist` for `ember-container-query`. To be safe, I'll set the value to `0` for now.